### ISSUE TITLE
`tryWrite` method?

### ISSUE THREAD

**Issue by aadityataparia** [uNetworking/uWebSockets.js#125]
Is there a `tryWrite` method like `tryEnd` or can there be?

difference with `tryEnd` being it won't require size argument when writing.

**Comment by alexhultman** [comment]
Try write is like try end but without a pre known size yes (content length vs chunked encoding)

**Comment by alexhultman** [comment]
Oh wait, there is no try write. You have to write smaller chunks like 16kb or 4kb.

**Comment by aadityataparia** [comment]
But can there be one, it would be useful IMO.

**Comment by alexhultman** [comment]
Yes it can be added but I haven't thought about how. You can send smaller chunks for now
